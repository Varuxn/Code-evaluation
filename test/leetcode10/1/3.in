97 213
52 68
49 45
13 87
3 11
41 79
6 33
54 6
6 17
31 40
23 47
41 0
43 62
18 63
10 45
19 28
57 33
50 61
61 10
19 54
32 74
90 86
92 39
59 90
3 72
63 22
69 35
19 32
17 90
33 2
4 5
95 91
30 95
27 83
31 43
52 60
66 2
7 48
13 19
17 53
15 69
55 64
11 51
60 75
23 59
63 46
17 25
14 93
17 18
27 34
44 42
73 2
26 52
42 84
35 87
89 94
37 59
44 89
23 74
0 16
10 20
2 67
35 5
6 35
47 90
19 90
4 92
61 68
29 71
2 11
38 47
11 88
0 15
12 38
30 86
81 64
63 69
11 45
90 1
7 65
3 7
5 12
95 57
22 25
0 1
9 96
44 66
26 24
35 67
23 90
90 37
7 57
27 95
47 58
59 1
93 8
13 23
45 56
29 68
83 91
57 73
59 88
2 45
1 14
28 8
93 94
78 88
8 76
0 8
35 73
4 9
17 28
0 54
5 21
9 16
1 34
48 51
17 40
42 71
3 22
88 40
58 32
38 15
92 48
22 36
26 31
24 40
65 73
32 53
94 68
96 48
55 65
15 81
20 61
90 73
55 58
22 77
24 20
3 27
30 59
12 94
9 59
89 84
96 87
44 67
16 29
16 39
57 24
41 63
22 76
4 43
12 49
18 85
33 59
22 46
3 92
20 89
46 62
45 78
63 64
16 70
58 33
28 82
88 92
54 75
61 91
1 80
4 6
92 95
15 78
49 33
12 25
19 24
3 4
2 3
9 30
43 61
87 49
7 42
89 30
43 34
28 49
89 48
1 12
12 22
16 50
84 17
24 72
38 62
8 27
20 31
12 50
36 63
5 10
82 52
91 53
38 7
0 64
33 66
42 55
33 41
67 12
1 26
34 44
74 11
0 13
59 34
13 83
55 4
0 2
88 31
3 37
78 63
49 63
