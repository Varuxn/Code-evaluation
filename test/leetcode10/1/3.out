1 80 5 21 16 70 18 85 22 77 41 79 45 56 